nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -102,6 +102,7 @@ Costmap2DROS::Costmap2DROS(const std::string & name, tf2_ros::Buffer & tf)
 
   // we need to make sure that the transform between the robot base frame
   // and the global frame is available
+  auto sleep_dir = std::chrono::milliseconds(100);
```

Comment:
What's the `dir` for? `direction`, `directory`? ... I think I just got it . Is it `duration`? If so, then you've got a spelling mistake. Should probably be `sleep_dur`